schema_based_on_diagram.sql

```diff
@@ -0,0 +1,54 @@
+CREATE DATABASE patient_clinic;
+
+
+CREATE TABLE patients(id INT GENERATED ALWAYS AS IDENTITY NOT NULL PRIMARY KEY,
+                                                                           name VARCHAR(250) NOT NULL,
+                                                                                             date_of_birth DATE NOT NULL);
+
+
+CREATE TABLE treatments(id INT GENERATED ALWAYS AS IDENTITY NOT NULL PRIMARY KEY,
+                                                                             type VARCHAR(250) NOT NULL,
+                                                                                               name VARCHAR(250) NOT NULL);
+
+
+CREATE TABLE medical_histories(id INT GENERATED ALWAYS AS IDENTITY NOT NULL PRIMARY KEY,
+                                                                                    admitted_at TIMESTAMP NOT NULL,
+                                                                                                          patient_id INT REFERENCES patients(id),
+                                                                                                                                    status VARCHAR(250) NOT NULL);
+
+
+CREATE TABLE invoices(id INT GENERATED ALWAYS AS IDENTITY NOT NULL PRIMARY KEY,
+                                                                           total_amount DECIMAL(8, 2) NOT NULL,
+                                                                                                      generated_at TIMESTAMP NOT NULL,
+                                                                                                                             payed_at TIMESTAMP NOT NULL,
+                                                                                                                                                medical_history__id INT NOT NULL REFERENCES medical_histories(id));
+
+
+CREATE TABLE invoice_items(id INT GENERATED ALWAYS AS IDENTITY NOT NULL PRIMARY KEY,
+                                                                                unit_price DECIMAL(8, 2) NOT NULL,
+                                                                                                         quantity INT NOT NULL,
+                                                                                                                      total_price DECIMAL(8, 2) NOT NULL,
+                                                                                                                                                invoice_id INT REFERENCES invoices(id),
+                                                                                                                                                                          treatment_id INT REFERENCES treatments(id));
+
+
```

Comment:
![image](https://github.com/Abdulwhbhussain/Vet-clinic/assets/93039359/b0151b9e-bfc5-4038-9799-a26adac25400)

- [ ] Great work so far on the project 👍 but it would be nice if you would see this issue which is very critical that is you have a license link here that takes us to an external website and there we find a template of the license 🤔 it means that your project is not licensed 😨 Kindly add a dedicated license file in your repo and link this MIT link to that file to make your project licensed 😊

**Hint: You can follow these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to add a dedicated license file in your repo to make your project licensed 👍**

Reply:
Added